### PR TITLE
test

### PR DESCRIPTION
**Pull request by zhannagrinko**
Just a test, space was added to the recipes.component.html

### CODE REVIEW

File: src/app/components/recipes/recipes.component.html [hunk 1]
```diff
@@ -17,4 +17,5 @@ <h2>{{selectedRecipe.name | uppercase}} Details</h2>
     </label>
   </div>
  
+
```
Comment (zhannagrinko) [inline src/app/components/recipes/recipes.component.html]: very informative and necessary change the project! Trust me!

Comment (zhannagrinko) [reply src/app/components/recipes/recipes.component.html]: Test reply

File: src/app/components/recipes/recipes.component.html [hunk 2]
```diff
@@ -17,4 +17,5 @@ <h2>{{selectedRecipe.name | uppercase}} Details</h2>
     </label>
```
Comment (zhannagrinko) [inline src/app/components/recipes/recipes.component.html]: One more comment

File: src/app/components/recipes/recipes.component.html (hunk 1 above)
Comment (svetlanaderyuga) [reply src/app/components/recipes/recipes.component.html]: :)

**Pull request closed without merging**